test/functional/test_framework/util.py

```diff
@@ -66,6 +66,11 @@ def assert_greater_than_or_equal(thing1, thing2):
         raise AssertionError("%s < %s" % (str(thing1), str(thing2)))
 
 
+def assert_true(thing, message = None):
+    if thing is not True:
+        msg = "%s it is not True" if message is None else message
```

Comment:
`it` needs to be removed from this sentence, also `% str(thing)` needs to be used to perform formatting so that the value of `thing` ends up in the message.